app/routes.py

```diff
@@ -1,2 +1,202 @@
-from flask import Blueprint
+from werkzeug.datastructures import Authorization
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, jsonify, make_response
+from datetime import datetime
+import os
+import requests
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("/<task_id>", methods=["GET","PUT", "DELETE"])
+def get_single_task(task_id):
+
+    task = Task.query.get(task_id)
+    # With the GET, POST and DELETE request if there is nothing we output this
+    if request == None or task == None:
+        return jsonify(None), 404
+    # This portion is the GET request for only one task
+    elif request.method == "GET":
+        return {"task": task.to_json()}, 200
+    elif request.method == "PUT":
+        # This portion is the PUT request for only one task
+        request_body = request.get_json()
+        task.title = request_body["title"]
+        task.description = request_body["description"]
+        db.session.commit()
+        return {"task": task.to_json()}, 200
+    elif request.method == "DELETE":
+        db.session.delete(task)
+        db.session.commit()
+        return {
+            "details": f"Task {task.task_id} \"{task.title}\" successfully deleted"
+            }, 200
+
+@tasks_bp.route("", methods=["GET"])
+def tasks_index():
+    
+    query_sorted = request.args.get("sort")
+    if query_sorted == "asc":
+        # Found in SQLALchemy documentation. 
+        # The order_by method takes the data in the user table (Task) 
+        # and filters by title in ascending order
+        tasks = Task.query.order_by(Task.title.asc())
+    elif query_sorted == "desc":
+        # Found in SQLALchemy documentation. 
+        # The order_by method takes the data in the user table (Task) 
+        # and filters by title in descending order
+        tasks = Task.query.order_by(Task.title.desc())
+    else:
+        tasks = Task.query.all()
+    # This portion is just a GET request
+    if tasks == None:
+        return []
```

Comment:
good check, but `all()` will already return an empty list if it doesn't find any tasks! No need for the if statement